app-emulation/cri-o/cri-o-1.9.1.ebuild

```diff
@@ -51,7 +62,7 @@ src_install() {
 	dodir   /etc/crio
 	insinto /etc/crio
 	doins   "${FILESDIR}/crio.conf"
-	doins   "${FILESDIR}/seccomp.json"
+	if use seccomp; then doins "${FILESDIR}/seccomp.json"; fi
```

Comment:
this could also be written as `use seccomp && doins "${FILESDIR}/seccomp.json"` which is a tad more succinct.

Reply:
> this could also be written as `use seccomp && doins "${FILESDIR}/seccomp.json"` which is a tad more succinct.

Done in d175d6d →0dd1b08.